src/redux/books/bookSlice.js

```diff
@@ -0,0 +1,29 @@
+const ADD_BOOK = 'bookstore/books/addBook';
+const REMOVE_BOOK = 'bookstore/books/removeBook';
+
+const initialState = {
+  books: [],
+};
+
+// Add Reducers
+export default function booksReducer(state = initialState, action) {
+  switch (action.type) {
+    case ADD_BOOK:
+      return { ...state };
+    case REMOVE_BOOK:
+      return {
+        ...state,
+        items: state.items.filter((item) => item.id !== action.id),
+      };
+    default:
+      return state;
+  }
+}
+
+export const addBook = () => ({
+  type: ADD_BOOK,
+});
+
+export const removeBook = () => ({
+  type: REMOVE_BOOK,
+});
```

Comment:
- [ ] Kindly make sure to provide the `id` of the book which is going to be removed as a parameter so the state will easily identify the book to be removed once this `removeBook` action is dispatched.

   - Suggestion:
      `export const removeBook = (id) => ({
          type: REMOVE_BOOK,
          id
      })`